template/dockerfiles/gomodule/Dockerfile

```diff
@@ -9,6 +7,10 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -v -o app-binary
 
 FROM gcr.io/distroless/static-debian12
+
+ENV PORT={{.PORT}}
```

Comment:
new syntax for key/value (old version through a legacy warning) - this needs to be in the final image, as env isn't  carried from builder